### DISCUSSION TITLE
is the fixed version of webview2 completely isolated from any edge updates?

### DISCUSSION THREAD

**Discussion by emaayan** [MicrosoftEdge/WebView2Feedback#4233] [Q&A]
if we are bundling a fixed version of webview2 is it possible that any of it's compoenents will be updated with an oraganization's windows update? 
we've seen strange issues in our which seems to randomly start issuing the same call many times in the last month,

**Comment by victorhuangwq** [comment]
If you are using a fixed version of webview2, then your application shouldn't be affected by the evergreen webview2 runtime update.

**Reply by emaayan** [reply]
how do i know forsure i'm using the evergreen or the fixed version cause i'm seeing i have the Microsoft.Web.WebView2.Core.dll included 
under runtimes

**Reply by victorhuangwq** [reply]
You can use https://github.com/david-risney/WebView2Utilities to figure out which runtime version you are using in the app. If it's not the same version number as the stable WebView2 runtime, it's likely it's a fixed runtime.
